spec/test-data/translations/glotpress-download/stubs/fr.xml

```diff
@@ -30,6 +31,7 @@ Language: fr
 		<item>En cours…</item>
 		<item>Toutes les commandes...</item>
 	</string-array>
+	<string name="stringref">\@string/shipping_label_woo_discount_bottomsheet_title</string>
```

Comment:
Intentional use of `\@string/` used on a `string` node of this GlotPress export stub, to reflect that GlotPress indeed escapes those in its exports, and to thus be able to test that we detect and auto-fix those during `quick_lint`